Replication Project_Final.Rmd

```diff
@@ -0,0 +1,860 @@
+in previous work and is therefore included in the regressions below.[...] A dummy variable is also introduced in the regression models to control for the municipalities located in the state of Mexico. Although some of these municipalities are technically rural, they are located in the Valley of Mexico in close proximity to Mexico City and tend to have higher homicide rates compared to the rest of the country, even when accounting for basic structural factors. 
+
+Finally, as described above, a measure of the local topography is used to test the effect of access to state institutions on the rate of homicidal violence [the standard deviation of elevation in a municipality]. However, remote rural areas are not only farther from the reach
+of state institutions, they are also farther from medical services. [...] I therefore introduce the number of medical doctors in the municipality per 10,000 residents according to the 1990 census as a predictor of homicide rates in all the regression models.""
+
+*Data were added to the dataset for analysis in order of data source - i.e. all data from the standard census first, then population density (which requires map data about the area of municipalities), then homicides, then the dummy variable. Data drawn from the Agricultural Census were added next, as they required matching by name instead of municipality code due to the format of the raw data. Once these data were combined into the main analytical dataframe, all relevant variables were computed, and then elevation data was added. Data regarding the state of Oaxaca were separated out through this process, as this state counts municipalities differently - each town is counted as its own municipality, which would skew the data. As noted above, Oaxacan municipalities are typically grouped into 30 districts for analysis--we completed this operation after creating the main analytical dataframe, recalculated all variables for the Oaxacan districts, and merged the two dataframes by municipality code.*
+
+
+```{r, warning=FALSE, error=FALSE}
+library(ggplot2)
+library(dplyr)
+library(xlsx)
+library(zoo)
+library(foreign)
+library(tidyr)
+library(gdata)
+library(maptools)
+library(gpclib) 
+library(RColorBrewer)
+library(scales)
+library(SDMTools)
+```
+
+**Municipalities, Indigenous Language-Speakers, Illiteracy**
+```{r, warning=FALSE, error=FALSE}
+#load main 1990 censo database from inegi to the town level
+censo90 <- read.table("data/censo_1990_37_var.txt", header = TRUE, sep = "\t", encoding = "latin1")
+
+#create table with variables of interest and population
+censo.1 <- tbl_df(censo90) %>%
+  filter(mun !=0 ) %>%
+  select(state = entidad, mun, twn = loc, total.pop = p_total, no.literacy1 = analfbet,
+         no.literacy2 = pob_no_l, n_hab_esp, habla_esp, state_name = nom_ent, mun_name = nom_mun) %>%
```

Comment:
This is very practical - I didn't know you could change variable names within the select function!